docs/guides/migration-guide-oauth.md

```diff
@@ -0,0 +1,240 @@
+---
+page_title: "Confluent Provider OAuth Authentication: Migration Guide"
+---
+# Confluent Provider OAuth Authentication: Migration Guide
+
+## Summary
+
+This guide is intended to help with the migration process and focuses only on the changes necessary to migrate from API key/secret to OAuth authentication in Confluent Provider.
```

Comment:
```suggestion
This guide is intended to help with the migration process and focuses only on the changes necessary to migrate from API key/secret (both Cloud and resource-specific) to OAuth authentication in Confluent Provider.
```

or something to match https://docs.confluent.io/cloud/current/api.html#section/Authentication/API-keys